src/components/SignIn.tsx

```diff
@@ -17,6 +17,19 @@ import iconDiscord from '../assets/images/icon-discord.png';
 import iconTwitch from '../assets/images/icon-twitch.png';
 import iconCoinbase from '../assets/images/icon-coinbase.png';
 
+import { EthereumPrivateKeyProvider } from "@web3auth/ethereum-provider";
+
+const chainConfig = {
+  chainNamespace: CHAIN_NAMESPACES.EIP155,
+  chainId: "0x1",
```

Comment:
@MayukhSamadder2000 chain ID is 1 / mainnet but you are using https://goerli.etherscan.io ?